api/src/controllers/replyController.ts

```diff
@@ -26,8 +27,10 @@ export const postReply = async (req: Request, res: Response) => {
   const snippets: RequestSnippetBlot[] = req.body.snippets;
   const delta: any = req.body.delta;
   const mentionedUserIds: number[] = req.body.mentionedUserIds;
+  const userId: number | undefined = req.userId;
   const { updatedReplyMessage, snippetEntities } = await createSnippetEntitiesFromReplyMessage(replyMessage, snippets);
   const thread: Thread | null = await ThreadRepository.findThreadById(threadId);
+  const projectId: number = parseInt(req.params.projectId as string);
```

Comment:
Url Params comes as string. Here it is unknown at first, because we didn't define the Request completely. But ig unless we have a middleware that parses `project` to integer, we have to do this in every controller.

I touched these again in webapp PR.